Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -464,6 +464,36 @@ public extension UIView {
 		}
 	}
 
+    /// SwifterSwift: Anchors all sides to superview's safe area.
+    ///
+    /// 1. On iOS 11 and latter anchors view's sides to superview's safe area
+    /// 2. On iOS 9-11 anchors leading and trailing to superview,
+    /// top and bottom to viewController's bottomLayoutGuide and topLayoutGuide.
```

Comment:
Please add backticks (\`) around `bottomLayoutGuide` and `topLayoutGuide`

Reply:
done